sass_processor/processor.py

```diff
@@ -107,7 +124,7 @@ def is_latest(self, sourcemap_filename):
         if not sourcemap_file or not os.path.isfile(sourcemap_file):
             return False
         sourcemap_mtime = os.stat(sourcemap_file).st_mtime
-        with open(sourcemap_file, 'r') as fp:
+        with io.open(sourcemap_file, 'r') as fp:
```

Comment:
Do we really need to use the `io` module here?

Reply:
No, I'll remove it.